simple/src/main/g8/test/Test$name__Camel$.tdml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+$!
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements. See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+!$
+
+<tdml:testSuite suiteName="$name;format="Camel"$" description="$name;format="Camel"$ tests"
```

Comment:
Does IBM's TDML processor use suiteName and description?  Currently I tend to leave them out because Daffodil's TDML processor ignores them.

Reply:
I'm not sure how IBM uses them, but I'm pretty sure they are optional. I have no opinion one way or another about keeping them. I imagine some day it might be useful to have them. For example, maybe one day we'll have a way to autogenerate tests based on just reading the .tdml files rather than also needing a .scala file. In that cas, the suiteName or description might be useful for generating those tests? I"m not really sure. Seems like good practice to have maybe?